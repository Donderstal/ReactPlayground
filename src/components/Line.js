import React from 'react'
import { View } from 'react-native'

export default Line = () => {
    return (
        <View style={{
            borderBottomColor: 'white',
            borderBottomWidth: 1,
            width: '100%',
            marginTop: '2,5%'}}/>
    )
}
