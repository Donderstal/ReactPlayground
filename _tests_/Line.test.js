// __tests__/Intro-test.js
import React from 'react';
import renderer from 'react-test-renderer';

import Line from '../src/components/Line';

test('renders correctly', () => {
  const tree = renderer.create(<Line />).toJSON();
  expect(tree).toMatchSnapshot();
});