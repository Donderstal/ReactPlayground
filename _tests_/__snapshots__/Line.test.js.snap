// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "white",
      "borderBottomWidth": 1,
      "marginTop": "2,5%",
      "width": "100%",
    }
  }
/>
`;
